features/desktop/post_preview.feature

```diff
@@ -16,26 +16,27 @@ Feature: preview posts in the stream
     Scenario: preview and post a text-only message
       Given I expand the publisher
       When I write the status message "I am eating yogurt"
-      And I press "Preview"
-      Then "I am eating yogurt" should be post 1
-      And the first post should be a preview
+      And I preview the post
+      Then I should see "I am eating yogurt" in the preview
 
+      Given I edit the post
       When I write the status message "This preview rocks"
-      And I press "Preview"
-      Then "This preview rocks" should be post 1
-      And I should not see "I am eating a yogurt"
+      And I preview the post
+      Then I should see "This preview rocks" in the preview
+      And I should not see "I am eating a yogurt" in the preview
 
+      Given I edit the post
       When I write the status message "I like rocks"
       And I press "Share"
       Then "I like rocks" should be post 1
-      And I should not see "This preview rocks"
+      When I expand the publisher
+      Then I should not be in preview mode
 
     Scenario: preview a very long message
       Given I expand the publisher
       When I insert an extremely long status message
-      And I press "Preview"
-      Then the preview should not be collapsed
-
+      And I preview the post
+      # Then the previewed should be collapsed
```

Comment:
I think it would be great to keep the test.
`Then the preview should not be collapsed`